valora-inc/wallet
# Remove the light client option

### Description

First step in removing the light client. All users will now be switched to using Forno (which has been the default for some time now). 

This was accelerated by the need to be compatible with the Espresso hard fork.
The Espresso hard fork will be activated on [Feb 8th 2022](https://forum.celo.org/t/espresso-on-mainnet-celo-blockchain-v1-5-0-release/2682).

I initially tried to quickly update the light client but faced [a few issues](https://valora-app.slack.com/archives/CL7BVQPHB/p1643030802027500).

We'll be able to remove more code, but here I just wanted to make the switch so we can cut v1.25.

### Tested

1. Install Valora <= 1.24
2. Create an account
3. Disable Data Saver in the setting
4. User can send transactions successfully (uses the light client under the hood)
5. Install Valora >= 1.25
6. See the Data Saver option is not available in the settings
7. User can still send transactions successfully (uses Forno nodes under the hood)

**Settings (before/after)**

<img src="https://user-images.githubusercontent.com/57791/150979127-2bf3024a-d058-4394-85cc-f9d7878c40fb.png"  width="50%" /><img src="https://user-images.githubusercontent.com/57791/150979062-d412fdc7-7da9-4b12-a804-9015abcfc762.png"  width="50%" />

### How others should test

See above.

### Related issues

- #1515

### Backwards compatibility

Yes
